packages/block-library/src/embed/block.json

```diff
@@ -43,12 +43,29 @@
 	"supports": {
 		"align": true,
 		"spacing": {
-			"margin": true
+			"margin": true,
+			"padding": true
```

Comment:
This applies the padding to the Embed block's wrapper. 

The justification for adding the padding support was so that it could be used to counter border radius and prevent the content from sticking out beyond the border.

This might need to be applied to the inner embed wrapper as well.